packages/front-end/components/Experiment/ImportExperimentList.tsx

```diff
@@ -139,10 +139,15 @@ const ImportExperimentList: FC<{
           {changeDatasource && supportedDatasources.length > 1 ? (
             <SelectField
               value={data.experiments.datasource}
-              options={supportedDatasources.map((d) => ({
-                value: d.id,
-                label: `${d.name}${d.description ? ` — ${d.description}` : ""}`,
-              }))}
+              options={supportedDatasources.map((d) => {
+                const isDefaultDataSource = d.id === defaultDataSource;
+                return {
+                  value: d.id,
+                  label: `${d.name}${
+                    d.description ? ` — ${d.description}` : ""
+                  } ${isDefaultDataSource ? " (default)" : ""}`,
```

Comment:
I'd love this `(default)` to be muted, italicized, etc. but from checking around adding that within-option styling to the SelectField seems non-trivial so probably not worth it.

Reply:
I feel the same way - I started down that path but I don't think the juice is worth the squeeze in this scenario.